docs/_docs/bestpractices/compute/compute.md

```diff
@@ -0,0 +1,168 @@
+author: arclares
+ms.date: 08/16/2024
+ms.service: finops
+description: 'Discover essential FinOps best practices to optimize cost efficiency and governance for your Azure resources.'
+
+---
+
+<span class="fs-9 d-block mb-4">Compute</span>
+Discover essential FinOps best practices to optimize cost efficiency and governance for your Azure compute resources.
+{: .fs-6 .fw-300 }
+
+[Share feedback](#️-looking-for-more){: .btn .fs-5 .mb-4 .mb-md-0 .mr-4 }
+
+<details open markdown="1">
+   <summary class="fs-2 text-uppercase">On this page</summary>
+
+- [🖥️ Virtual Machines](#virtual-machines)
+- [🖱️ Azure Kubernetes Service](#azure-kubernetes-service)
+- [🙋‍♀️ Looking for more?](#️-looking-for-more)
+- [🧰 Related tools](#-related-tools)
+
+</details>
+
+---
+
+## Azure Kubernetes Service
+
+### Query: AKS Cluster
+
+This Azure Resource Graph (ARG) query retrieves detailed information about Azure Kubernetes Service (AKS) clusters within your Azure environment. 
+
+#### Category
+
+Resource management
+
+#### Query
+
+<details>
+  <summary>Click to view the code</summary>
+  <div class="code-block">
+    <pre><code>resources
+  | where type == "microsoft.containerservice/managedclusters"
+  | extend AgentPoolProfiles = properties.agentPoolProfiles
+  | mvexpand AgentPoolProfiles
+  | project
+      id,
+      ProfileName = tostring(AgentPoolProfiles.name),
+      Sku = tostring(sku.name),
+      Tier = tostring(sku.tier),
+      mode = AgentPoolProfiles.mode,
+      AutoScaleEnabled = AgentPoolProfiles.enableAutoScaling,
+      SpotVM = AgentPoolProfiles.scaleSetPriority,
+      VMSize = tostring(AgentPoolProfiles.vmSize),
+      nodeCount = tostring(AgentPoolProfiles.['count']),
+      minCount = tostring(AgentPoolProfiles.minCount),
+      maxCount = tostring(AgentPoolProfiles.maxCount),
+      location,
+      resourceGroup,
+      subscriptionId,
+      AKSname = name
+</code></pre>
+  </div>
+</details>
+
+<br>
+
+## Virtual machines
+
+### Query: List Virtual Machines stopped (and not deallocated)
+
+This Azure Resource Graph (ARG) query identifies Virtual Machines (VMs) in your Azure environment that are not in the 'deallocated' or 'running' state. It retrieves details about their power state, location, resource group, and subscription ID.
+
+#### Category
+
+Waste reduction
+
+#### Query
+
+<details>
+  <summary>Click to view the code</summary>
+```kql resources 
+    | where type =~ 'microsoft.compute/virtualmachines' 
+        and tostring(properties.extended.instanceView.powerState.displayStatus) != 'VM deallocated' 
+        and tostring(properties.extended.instanceView.powerState.displayStatus) != 'VM running'
+    | extend PowerState=tostring(properties.extended.instanceView.powerState.displayStatus)
+    | extend VMLocation=location
+    | extend resourceGroup=strcat('/subscriptions/',subscriptionId,'/resourceGroups/',resourceGroup)
+    | order by id asc
+    | project id, PowerState, VMLocation, resourceGroup, subscriptionId
+```

```

Comment:
Markdown should be indented to the same level as the sibling content. The code should also be at the same level (except for indented code, of course).
````suggestion
  ```kql
  resources 
  | where type =~ 'microsoft.compute/virtualmachines' 
      and tostring(properties.extended.instanceView.powerState.displayStatus) != 'VM deallocated' 
      and tostring(properties.extended.instanceView.powerState.displayStatus) != 'VM running'
  | extend PowerState=tostring(properties.extended.instanceView.powerState.displayStatus)
  | extend VMLocation=location
  | extend resourceGroup=strcat('/subscriptions/',subscriptionId,'/resourceGroups/',resourceGroup)
  | order by id asc
  | project id, PowerState, VMLocation, resourceGroup, subscriptionId
  ```
````